README.md

```diff
@@ -79,3 +79,22 @@ Run with coverage `python -m pytest --cov=src`
 Run with coverage and missing lines `python -m pytest --cov-report term-missing --cov=src`
 
 Run with coverage, missing lines and junit output `python -m pytest --cov-report term-missing --junitxml="result.xml" --cov=src`
+
+## Yara rules checks.
+
+There is a Jenkins job run on a schedule from the Jenkinsfile-check-rules file. This carries out the following steps. 
```

Comment:
Maybe reference the name of the Jenkins job.

Reply:
I've put the name in and a link to the job.